<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Su Tech Smart Surveilliance Dashboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>636</width>
             <height>422</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QGridLayout" name="gridLayout_4"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Cameras</string>
         </attribute>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1500</width>
            <height>71</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="columnCount">
           <number>10</number>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">6</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">7</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">8</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">9</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">10</string>
           </property>
          </column>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Recorded Content</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Logs</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File Path / RTSP Url / Camera Index :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="videoEdit">
          <property name="text">
           <string>./demo.mp4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="text">
           <string>Add Camera</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
